BookPlayer/Coordinators/SpeedCoordinator.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  StorageCoordinator.swift
+//  BookPlayer
+//
+//  Created by Gianni Carlo on 29/9/21.
+//  Copyright © 2021 Tortuga Power. All rights reserved.
+//
+
+import Foundation
+import BookPlayerKit
+
+class SpeedCoordinator: Coordinator {
+  let libraryService: LibraryServiceProtocol
+  let playerManager: PlayerManagerProtocol
+
+  init(libraryService: LibraryServiceProtocol,
+       playerManager: PlayerManagerProtocol,
+       navigationController: UINavigationController) {
+    self.libraryService = libraryService
+    self.playerManager = playerManager
+
+    super.init(navigationController: navigationController, flowType: .modal)
+  }
+
+  override func start() {
+      let vc = SpeedViewController.instantiate(from: .Player)
+      let viewModel = SpeedViewModel(playerManager: self.playerManager, libraryService: self.libraryService)
+      viewModel.coordinator = self
+      vc.viewModel = viewModel
+      
+      let nav = AppNavigationController.instantiate(from: .Main)
```

Comment:
Regarding the white line that you mention, it looks that it's due to this NavigationController. Since we won't be pushing another VC after this, I think we can present directly the SpeedVC 🤔, the dismissal should work fine

Reply:
here there is a problem with the transition, if I do this, the NavigationController is lost and the modal is shown in the library.
found this way
```
nav.interactivePopGestureRecognizer!.state = .began
```

Reply:
done